### ISSUE TITLE
Update for V3

### ISSUE THREAD

**Issue by moccassins** [pemedia/payload-visual-editor#38]
Could some one update the plugin to be compatible with payload v3?

**Comment by dkirchhof** [comment]
Since payload 3 is still in beta, I didn't have time to investigate the new version. In conclusion there wasn't time to migrate the library as well. Sorry.

**Comment by roybarber** [comment]
Looks like payload v3 comes with a preview mode.

**Comment by averyhere** [comment]
+1 - Payload v3 comes with a preview mode, but you have to get the enterprise plan to get access to the visual editor. Would love an open source solution like this.

**Comment by zatakiya** [comment]
Now that payload v3 is stable, any plan to make it compatible with it ? If yes, what could be estimated time of arrival ?

**Comment by dkirchhof** [comment]
Actually I didn't update any project to Payload 3, so I wasn't able to check out the integrated live preview.
But due to the docs, I would recommend to use the built-in server-side preview. Our plugin only works client-side.

So if there aren't any significant reasons to use our plugin instead, it is unlikely that we will update it to Payload 3. Sorry.

**Comment by zhyc9de** [comment]
+1

**Comment by SalahAdDin** [comment]
> +1 - Payload v3 comes with a preview mode, but you have to get the enterprise plan to get access to the visual editor. Would love an open source solution like this.

It should mean the main team hired this team to work on the preview mode, i. e., this plugin could be deprecated.

**Comment by Chetan-svg** [comment]
Any plans to update if for V3??

**Comment by tan-ahmed** [comment]
bump

**Comment by SalahAdDin** [comment]
> Any plans to update it for V3??

I don't think so, Payload CMS 3 already has almost all of these package features for pay.
